src/components/button.js

```diff
@@ -0,0 +1,46 @@
+import React,  { Component, } from "react";
+import PropTypes from "prop-types";
+import { Button, View, TouchableOpacity, Text} from 'react-native';
+import styled from 'styled-components';
+
+class GeneralButton extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+    <CustomButton type="submit" backgroundColor={this.props.buttonColorBackground} >
+      <ButtonText>
+        {this.props.children}
+      </ButtonText>
+    </CustomButton>
+    );
+  }
+}
+
+// STYLED-COMPONENTS
+const CustomButton = styled.TouchableOpacity `
+    border-radius: 12px;
+    width: 350px;
+    padding-horizontal: 15px;
+    padding-vertical: 18px;
+    background: ${props => props.backgroundColor};
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/Airbenders-490/AirbendersFE/pullRequest?prid=8247201)
```suggestion
    background: ${(props) => props.backgroundColor};
```